### DISCUSSION TITLE
tutorials/animation/cutout_animation

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#191] [User-contributed notes]
# tutorials/animation/cutout_animation

What is it?: Traditionally, cutout animation is a type of stop motion animation in which pieces of paper (or other thin material) are cut into special shapes and arranged in two-dimensional represe...

https://docs.godotengine.org/en/stable/tutorials/animation/cutout_animation.html

**Comment by Szakal-raxid** [comment]
Inverse Kinematics (IK) in godot 4.3 are no longer under the bone icon, here`s a link to godot forum with the solution i`ve found.
https://forum.godotengine.org/t/4-3-ik-chains-option-missing-from-menu/87459/2

**Comment by djkris105** [comment]
Ugh I can't seem to make the correct bones. First of all, I don't see all the options listed here, I only see the "Make bones" option when I click over "skeleton options". Then, when I create that it creates a bunch of bone nodes between different nodes and none of this is present in this tutorial. Help me?

**Reply by LaksahanWijerathne** [reply]
I have same issue like this. I have followed both tutorials about bone making. but those not worked for me. but I have followed  exactly what they said. In this one I don't see the option that mention here while making bones.

**Comment by elcarn23** [comment]
I'm following the tutorial as well in 4.3 and when I get to the create bones step for the right arm for an example and when I select them from the end to the beginning two things happen.  1. It rearranges my nodes to be arm_r -> end -> hand_r.  and 2. It gives me 3 bones none of which are connected and are just represented by a small arrow pointed to the right from each bone.  Is there an update to how to do cutout animation?  I'm new to Godot so if I've missed something I'm sorry.

**Reply by DebugNugget** [reply]
i found this tutorial works pretty well: https://www.youtube.com/watch?v=GQLL7laFqQY&t=1s
but it is a more manual aproach

**Reply by Wayne-Kim** [reply]
⬆️⬆️⬆️⬆️⬆️⬆️⬆️⬆️⬆️⬆️⬆️⬆️⬆️⬆️⬆️⬆️⬆️⬆️⬆️⬆️⬆️⬆️⬆️⬆️⬆️⬆️⬆️
The video above is more useful than this docs.

**Comment by themipper** [comment]
In 4.4 there is only the option "make Bone2D node(s) from node(s)" - all the other options are missing. And the result is not a new root node consisting of a hierarchy of Bone2D. Instead it inserts Bone2D-nodes as root nodes in the existing hierarchy of nodes. And that hierarchy doesn't work with a skeleton2D as only Bone2D or Skeleton2D are allowed within the hierarchy. 

Am I missing something on how to use this function?
Have the original menu items moved somewhere else?

**Comment by wjtaylor-games** [comment]
Instead of using `RemoteTransform2D`, why not just change the Z index under Ordering for each sprite?

**Comment by Wayne-Kim** [comment]
Even after following this document, I was unable to configure Skeletons. I suspect this document has not been updated to suit the latest version. Therefore, I hope no one else wastes their time like I did until it is updated

**Reply by Calinou** [reply]
Indeed, this documentation is currently outdated as per https://github.com/godotengine/godot-docs/issues/6388. Help is welcome on updating it :slightly_smiling_face:

**Reply by Wayne-Kim** [reply]
I'm still in the learning phase with Godot and haven't even properly applied skeletons to my app yet. So, updating the documentation is difficult for me right now. In fact, it would probably make the documentation even more complicated at this point. I'll give it a try once I'm a bit more familiar
